### PR TITLE
More Cupertino dialog UI tweaks

### PR DESCRIPTION
**Pull request by xster**
#9776

### CODE REVIEW

File: packages/flutter/lib/src/cupertino/dialog.dart [hunk 1]
```diff
@@ -218,6 +224,9 @@ class CupertinoDialogAction extends StatelessWidget {
   Widget build(BuildContext context) {
     TextStyle style = _kCupertinoDialogActionStyle;
 
+    if(isDefault)
```
Comment (Hixie) [inline packages/flutter/lib/src/cupertino/dialog.dart]: nit: space after if

File: packages/flutter/lib/src/cupertino/dialog.dart [hunk 2]
```diff
@@ -200,6 +201,11 @@ class CupertinoDialogAction extends StatelessWidget {
   /// If this is set to null, the button will be disabled.
   final VoidCallback onPressed;
 
+  /// Set to true if button is the default choice in the dialog.
+  ///
+  /// Default buttons are bolded.
```
Comment (Hixie) [inline packages/flutter/lib/src/cupertino/dialog.dart]: This is an odd semantic. Does it not do anything else?

Maybe we should call it "recommended"? Does iOS call it "default"?

**Comment by Hixie** [comment]
LGTM

File: packages/flutter/lib/src/cupertino/dialog.dart (hunk 1 above)
Comment (xster) [inline packages/flutter/lib/src/cupertino/dialog.dart]: done

File: packages/flutter/lib/src/cupertino/dialog.dart (hunk 2 above)
Comment (xster) [inline packages/flutter/lib/src/cupertino/dialog.dart]: Apple seems to use the same wording https://developer.apple.com/ios/human-interface-guidelines/ui-views/alerts/

Renamed to isDefaultAction to be more specific. Renamed isDestructive to isDestructiveAction too for categorisation

**Comment by Hixie** [comment]
BTW, moving forward we have to not make changes like renaming properties. Part of being API stable and everything.

**Pull request merged**